frontend/src/main.ts

```diff
@@ -1,6 +1,29 @@
 import { createApp } from 'vue';
+import { createRouter, createWebHashHistory } from 'vue-router';
 import App from './App.vue';
 
 import './assets/main.css';
+//routes
+import GuidePageVue from './routes/guide/GuidePage.vue';
+import IndexPageVue from './routes/index/IndexPage.vue';
+import QuestionPageVue from './routes/question/QuestionPage.vue';
+import ResultPageVue from './routes/result/ResultPage.vue';
+import SummaryPageVue from './routes/summary/SummaryPage.vue';
 
-createApp(App).mount('#app');
+const routes = [
+  { path: '/', component: IndexPageVue },
+  { path: '/guide/:nr', component: GuidePageVue },
+  { path: '/question/:nr', component: QuestionPageVue },
```

Comment:
Guide a question budou určitě mít svoji url? Je to tak dohodnuto? Z designu jsem pochopil, že to je jen jedna strának a otázky jsou jen jako slider.

Reply:
Z hlediska flow a toho jak je ten view strukturovanej to dava smysl